### PR TITLE
[4.0 dev] classes generated in params causing issue in layout builder

### PR DESCRIPTION
**Pull request by ravisaxena23**
### Summary of Changes
tried to make layout builder free of class="column-count-md-2 column-count-lg-3 by using condition. 
1 Because of this whole screen cannot be used. it stuck all the stuff to left.
2 because of it blue buttons are working but changes are not shown on frontend.
before
Screenshot (45)

### Testing Instructions
please check for possible break this line can cause.

### Expected result
this line should not cause break in any other code.

### Actual result
working fine on my machine.
after
Screenshot (49)

### CODE REVIEW

**Comment by brianteeman** [comment]
How can you say that this change will not break any existing code

**Comment by ravisaxena23** [comment]
> How can you say that this change will not break any existing code

it make changes only when label is layout builder else it will make no change to code.hence everything should run as good as they were running before

**Comment by brianteeman** [comment]
I suggest that you read the code again. Because the code in the PR absolutely changes existing behaviour

image

**Comment by ravisaxena23** [comment]
> I suggest that you read the code again. Because the code in the PR absolutely changes existing behaviour
> 
> image

sorry my bad. made right again

**Comment by ravisaxena23** [comment]
made code right again

File: layouts/joomla/edit/params.php [hunk 1]
```diff
@@ -145,7 +145,7 @@
 		{
 			echo '<fieldset id="fieldset-' . $name . '" class="options-form ' . (!empty($fieldSet->class) ? $fieldSet->class : '') . '">';
 			echo '<legend>' . $label . '</legend>';
-			echo '<div class="column-count-md-2 column-count-lg-3">';
+			echo $label=="Layout Builder" ?'<div class="">':'<div class="column-count-md-2 column-count-lg-3">';
```
Comment (Quy) [inline layouts/joomla/edit/params.php]:
```suggestion
			echo $label === 'Layout Builder' ? '<div class="">' : '<div class="column-count-md-2 column-count-lg-3">';
```

Comment (Quy) [reply layouts/joomla/edit/params.php]: Why empty `class=""`?

**Comment by brianteeman** [comment]
I have no idea why you are doing this but as the value of the label is coming from

`		$label = Text::_($fieldSet->label);`

it is being translated and therefore your code wont work

**Comment by ravisaxena23** [comment]
> I have no idea why you are doing this but as the value of the label is coming from
> 
> ` $label = Text::_($fieldSet->label);`
> 
> it is being translated and therefore your code wont work

I don't know why but the code is working already tried it in pagebuilder repo.  i think when
$label = Text::_($fieldSet->label); is translated to "layout builder" string matching take place. in this block of code first $label value is set then condition is checked therefore it is working. my main aim is to restrict this <div class="column-count-md-2 column-count-lg-3"> coming in layout builder. since it is causing trouble. it was the best way as per my small knowledge. 

I'm open for suggestions please suggest me a better way @brianteeman .

**Comment by ravisaxena23** [comment]
@Quy removed class thanks for suggestion i forget to remove.

**Comment by brianteeman** [comment]
As soon as the label is translated or there is a language override then your code obviously wont work

1. You would need to compare with the constant not the value
2. Are you sure that reusing this params layout is the best option - wouldn't it be better to have your own layout to use for the layout builder

**Comment by ravisaxena23** [comment]
1 you are right I'm  sorry I forget joomla uses multi language I will work into it.
2 I don't think we should make new params layou for layout builder. We should never repeat our code. All work can be done by just adding one condition why should we make new layout? It will increase code. If still it is necessary please explain.

**Comment by ravisaxena23** [comment]
@brianteeman hey i am using id as a constant for layout builder. while defining fileldset for layout builder we will need id. what you say? am i in right direction please guide.

**Comment by brianteeman** [comment]
The entire point of layouts is that they are single purpose.

**Comment by ravisaxena23** [comment]
> The entire point of layouts is that they are single purpose.

ok

**Comment by Quy** [comment]
@ravisaxena23 Please advise status.

**Comment by ravisaxena23** [comment]
@Quy I'm trying to understand code more better. And thinking a way to make changes possible without changing any joomla core file. For now we can close this pr. I will try something new in pagebuilder repo. If it works we don't have to make changes in Joomla core.
I'm open for any suggestion and help. If there is any better way to do things please explain to me.

**Comment by Quy** [comment]
Let’s close for now. If you come up with a fix, this can be reopened. Thanks.

**Pull request closed without merging**